docs/contributor-guide.md

```diff
@@ -39,19 +39,6 @@ make test-unit
 make test-e2e
 ```
 
-## Run Helm Tests
-Run the helm test for a particular descheduler release by setting below variables,
-```
-HELM_IMAGE_REPO="descheduler"
-HELM_IMAGE_TAG="helm-test"
-HELM_CHART_LOCATION="./charts/descheduler"
-```
-The helm tests runs as part of descheduler CI. But, to run it manually from the descheduler root,
-
-```
-make test-helm
```

Comment:
Do we have an alternative to running the helm test locally? Can `.github/workflows/helm.yaml` by run the same way through `make test-helm`?

Reply:
I'm figuring out how to run ct locally without a lot of dependencies, will add that here :)

Reply:
Threw some things on Makefile, docs and ./hack/verify-helm.sh